app/code/Transoft/Blog/etc/adminhtml/routes.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:App/etc/routes.xsd">
+    <!--Use router 'standard' for frontend route-->
+    <router id="admin">
+        <!--Define a custom route with id and frontName-->
+        <route id="transoft_blog" frontName="transoft_blog">
```

Comment:
Why do we have different route name in frontend and adminhtml?

Reply:
fixed